packages/core/src/account/simple.ts

```diff
@@ -58,6 +59,26 @@ export class SimpleSmartContractAccount<
     });
   }
 
+  override async encodeBatchExecute(
+    _txs: BatchUserOperationCallData
+  ): Promise<`0x${string}`> {
+    const [targets, datas] = _txs.reduce(
+      (accum, curr) => {
+        accum[0].push(curr.target);
+        accum[0].push(curr.data);
```

Comment:
shouldn't this be `accum[1]` maybe I'm reading this wrong...

Reply:
No you're right, good catch